packages/neptune-extension/src/nbextension/utils/window.ts

```diff
@@ -36,8 +31,7 @@ export async function openNotebookInNewWindow(content: any) {
     type: 'notebook',
   };
 
-  await contentManager.save(newNotebook.path, notebook)
+  await contentManager.save(newNotebook.path, notebook);
 
-  w.location.href = url;
+  window.open(url, Jupyter._target);
```

Comment:
I made changes to the flow of opening new window to be consistent to how we do it in jupyter lab.